index.html

```diff
@@ -23,5 +24,152 @@ <h1 class="playfairFont">José Fernández-Caminero</h1>
   </main>
 
   <section class="middleDeco"></section>
+
+  <section>
+    <h2 class="playfairFont padding10twenty">Projects</h2>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you move both of your sections inside the the `main` tag for best semantics reasons and good practices.
```html
   ...
   <main>
      <section>
         Your code for the section 1 here.....
      <section>

     <section>
         Your code for the section 2 here.....
      <section>
    
     <section>
        code for your next section here....
     <section>
      .....
   </main>
    ....